task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class CustomStack
+    {
+        private const int ARRAY_LENGTH = 10;
```

Comment:
What is array? It is better to use smth like `INIT_LENGTH`.

Reply:
Done.